packages/cli/commands/config/list.js

```diff
@@ -0,0 +1,63 @@
+const { logger } = require('@hubspot/cli-lib/logger');
+const { getConfig } = require('@hubspot/cli-lib/lib/config');
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cli-lib');
+
+const {
+  addConfigOptions,
+  addAccountOptions,
+  addUseEnvironmentOptions,
+  getAccountId,
+  setLogLevel,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { logDebugInfo } = require('../../lib/debugInfo');
+const { validateAccount } = require('../../lib/validation');
+const { getTableContents, getTableHeader } = require('../../lib/table');
+
+const loadAndValidateOptions = async options => {
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath, options);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validateAccount(options)))) {
+    process.exit(1);
+  }
+};
+
+exports.command = 'list';
+exports.describe = 'List names of portals defined in config';
+
+exports.handler = async options => {
+  loadAndValidateOptions(options);
+
+  const accountId = getAccountId(options);
+
+  trackCommandUsage('functions-list', {}, accountId);
+
+  const config = getConfig();
+  const portalData = config.portals.reduce((acc, portal) => {
+    acc.push([portal.name, portal.portalId, portal.authType]);
```

Comment:
Wondering if `authType` should be an enum?

Reply:
We could definitely make it an `enum` though it's not elsewhere in the code. I think doing this may be out of scope for this PR though.

Is there something that you're worried about with not using an `enum` here?

Reply:
Ah, sorry I was thinking that we could validate that a correct authType exists but might not be a big deal